numpy/core/multiarray.py

```diff
@@ -15,7 +16,7 @@
 from numpy.core._multiarray_umath import *
 from numpy.core._multiarray_umath import (
     _fastCopyAndTranspose, _flagdict, _insert, _reconstruct, _vec_string,
-    _ARRAY_API, _monotonicity
+    _ARRAY_API, _monotonicity, _get_ndarray_c_version
```

Comment:
Fixes pymc-devs/pymc3#3340, sympy/sympy#16556, and probably some others.

Reply:
The name begins with a '_', shouldn't we be discouraging its use rather than adding more private methods to the public API?

Reply:
Perhaps, but removing it from the API broke downstream packages, and I don't think it was a deliberate choice.

Also, this member is even documented, so I think is intended to be "public but only for library use" or something.